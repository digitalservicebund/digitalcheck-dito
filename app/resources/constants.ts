export const PRE_CHECK_START_BUTTON_ID = "preCheck-start-button";

export const HIGHLIGHT_COLORS = {
  1: { background: "bg-principle-1", border: "border-principle-1" },
  2: { background: "bg-principle-2", border: "border-principle-2" },
  3: { background: "bg-principle-3", border: "border-principle-3" },
  4: { background: "bg-principle-4", border: "border-principle-4" },
  5: { background: "bg-principle-5", border: "border-principle-5" },
  6: { background: "bg-principle-6", border: "border-principle-6" },
} as const;
