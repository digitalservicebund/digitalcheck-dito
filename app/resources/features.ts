export const features: Record<string, string> = {};
