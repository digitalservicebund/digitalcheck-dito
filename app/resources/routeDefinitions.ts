// These routes can't be exported from ~/resources/routeDefinitions.ts as that file isn't part of the app environment: https://github.com/remix-run/react-router/issues/12392

const PRE_CHECK_PDF = "digitalcheck-vorpruefung.pdf";
const DOCUMENTATION_PDF = "digitalcheck-begleitende-dokumentation.pdf";

export type Route = {
  url: string;
  title: string;
  parent?: string;
};

export const ROUTE_LANDING: Route = {
  url: "/",
  title: "Startseite",
};
export const ROUTE_PRECHECK: Route = {
  url: "/vorpruefung",
  title: "Vorprüfung: Digitalbezug einschätzen",
  parent: ROUTE_LANDING.url,
};
export const ROUTE_GENERAL_INFO: Route = {
  url: `${ROUTE_PRECHECK.url}/hinweise`,
  title: "Allgemeine Hinweise",
  parent: ROUTE_PRECHECK.url,
};
export const ROUTE_RESULT: Route = {
  url: `${ROUTE_PRECHECK.url}/ergebnis`,
  title: "Ergebnis",
  parent: ROUTE_PRECHECK.url,
};
export const ROUTE_RESULT_PDF: Route = {
  url: `${ROUTE_RESULT.url}/${PRE_CHECK_PDF}`,
  title: "Vorprüfung als PDF",
  parent: ROUTE_RESULT.url,
};
export const ROUTE_METHODS: Route = {
  url: "/methoden",
  title: "Regelung erarbeiten",
  parent: ROUTE_LANDING.url,
};
export const ROUTE_METHODS_RESPONSIBLE_ACTORS: Route = {
  url: `${ROUTE_METHODS.url}/zustaendige-akteurinnen-auflisten`,
  title: "Akteure auflisten",
  parent: ROUTE_METHODS.url,
};
export const ROUTE_METHODS_TASKS_PROCESSES: Route = {
  url: `${ROUTE_METHODS.url}/ablaeufe-aufgaben-erfassen`,
  title: "Abläufe erfassen",
  parent: ROUTE_METHODS.url,
};
export const ROUTE_METHODS_COLLECT_IT_SYSTEMS: Route = {
  url: `${ROUTE_METHODS.url}/it-systeme-erfassen`,
  title: "IT-Systeme erfassen",
  parent: ROUTE_METHODS.url,
};
export const ROUTE_METHODS_FIVE_PRINCIPLES: Route = {
  url: `${ROUTE_METHODS.url}/fuenf-prinzipien`,
  title: "Fünf Prinzipien",
  parent: ROUTE_METHODS.url,
};
export const ROUTE_METHODS_TECHNICAL_FEASIBILITY: Route = {
  url: `${ROUTE_METHODS.url}/technische-umsetzbarkeit`,
  title: "Technische Umsetzbarkeit sicherstellen",
  parent: ROUTE_METHODS.url,
};

export const ROUTE_INTEROPERABILITY: Route = {
  url: "/interoperabel",
  title: "Interoperable Regelungen",
  parent: ROUTE_LANDING.url,
};
export const ROUTE_INTEROPERABILITY_SPOC: Route = {
  url: `${ROUTE_INTEROPERABILITY.url}/nationale-kontaktstelle`,
  title: "Nationale Kontaktstelle",
  parent: ROUTE_INTEROPERABILITY.url,
};

export const ROUTE_DOCUMENTATION: Route = {
  url: "/dokumentation",
  title: "Dokumentation",
  parent: ROUTE_LANDING.url,
};

export const ROUTE_DOCUMENTATION_STATIC_PDF: Route = {
  url: `/download/${DOCUMENTATION_PDF}`,
  title: "Begleitende Dokumentation als PDF",
};

export const ROUTE_SUPPORT: Route = {
  url: "/unterstuetzung",
  title: "Unterstützungsangebote",
  parent: ROUTE_LANDING.url,
};

export const ROUTE_IMPRINT: Route = {
  url: "/impressum",
  title: "Impressum",
  parent: ROUTE_LANDING.url,
};
export const ROUTE_PRIVACY: Route = {
  url: "/datenschutz",
  title: "Datenschutzerklärung",
  parent: ROUTE_LANDING.url,
};
export const ROUTE_A11Y: Route = {
  url: "/barrierefreiheit",
  title: "Barrierefreiheit",
  parent: ROUTE_LANDING.url,
};
export const ROUTE_SITEMAP: Route = {
  url: "/sitemap",
  title: "Sitemap",
  parent: ROUTE_LANDING.url,
};

export const ROUTE_EXAMPLES: Route = {
  url: "/beispiele",
  title: "Beispiele für Digitaltauglichkeit",
  parent: ROUTE_LANDING.url,
};
export const ROUTE_PRINCIPLES: Route = {
  url: `${ROUTE_EXAMPLES.url}/prinzipien`,
  title: "Prinzipien",
  parent: ROUTE_EXAMPLES.url,
};
export const ROUTE_PRINCIPLE_DIGITAL_COMMUNICATION = {
  url: `${ROUTE_PRINCIPLES.url}/digitale-kommunikation-sicherstellen`,
  title: "Prinzip 1 in Regelungstexten",
  parent: ROUTE_PRINCIPLES.url,
};
export const ROUTE_PRINCIPLE_REUSE_DATA_AND_STANDARDS = {
  url: `${ROUTE_PRINCIPLES.url}/wiederverwendung-von-daten-und-standards-ermoeglichen`,
  title: "Prinzip 2 in Regelungstexten",
  parent: ROUTE_PRINCIPLES.url,
};
export const ROUTE_PRINCIPLE_DATA_PROTECTION_AND_INFORMATION_SECURITY = {
  url: `${ROUTE_PRINCIPLES.url}/datenschutz-und-informationssicherheit-gewaehrleisten`,
  title: "Prinzip 3 in Regelungstexten",
  parent: ROUTE_PRINCIPLES.url,
};
export const ROUTE_PRINCIPLE_CLEAR_REGULATIONS = {
  url: `${ROUTE_PRINCIPLES.url}/klare-regelungen-fuer-eine-digitale-ausfuehrung-finden`,
  title: "Prinzip 4 in Regelungstexten",
  parent: ROUTE_PRINCIPLES.url,
};
export const ROUTE_PRINCIPLE_AUTOMATION = {
  url: `${ROUTE_PRINCIPLES.url}/automatisierung-ermoeglichen`,
  title: "Prinzip 5 in Regelungstexten",
  parent: ROUTE_PRINCIPLES.url,
};

export const ROUTE_REGELUNGEN: Route = {
  url: `${ROUTE_EXAMPLES.url}/regelungen`,
  title: "Gesetze",
  parent: ROUTE_EXAMPLES.url,
};

export const ROUTE_VISUALISATION: Route = {
  url: `${ROUTE_EXAMPLES.url}/visualisierung`,
  title: "Visualisierung",
  parent: ROUTE_EXAMPLES.url,
};

export const ROUTE_VISUALISATIONS: Route = {
  url: `${ROUTE_EXAMPLES.url}/visualisierungen`,
  title: "Visualisierungen",
  parent: ROUTE_EXAMPLES.url,
};

export const ROUTES: Route[] = [
  ROUTE_LANDING,
  ROUTE_PRECHECK,
  ROUTE_GENERAL_INFO,
  ROUTE_RESULT,
  ROUTE_METHODS,
  ROUTE_METHODS_RESPONSIBLE_ACTORS,
  ROUTE_METHODS_TASKS_PROCESSES,
  ROUTE_METHODS_COLLECT_IT_SYSTEMS,
  ROUTE_METHODS_FIVE_PRINCIPLES,
  ROUTE_METHODS_TECHNICAL_FEASIBILITY,
  ROUTE_INTEROPERABILITY,
  ROUTE_INTEROPERABILITY_SPOC,
  ROUTE_SUPPORT,
  ROUTE_DOCUMENTATION,
  ROUTE_IMPRINT,
  ROUTE_PRIVACY,
  ROUTE_A11Y,
  ROUTE_SITEMAP,
  ROUTE_EXAMPLES,
  ROUTE_PRINCIPLES,
  ROUTE_PRINCIPLE_DIGITAL_COMMUNICATION,
  ROUTE_PRINCIPLE_REUSE_DATA_AND_STANDARDS,
  ROUTE_PRINCIPLE_DATA_PROTECTION_AND_INFORMATION_SECURITY,
  ROUTE_PRINCIPLE_CLEAR_REGULATIONS,
  ROUTE_PRINCIPLE_AUTOMATION,
  ROUTE_VISUALISATIONS,
];
