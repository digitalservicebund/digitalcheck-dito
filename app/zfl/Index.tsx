import FactCheckOutlinedIcon from "@digitalservicebund/icons/FactCheckOutlined";
import WidgetsOutlinedIcon from "@digitalservicebund/icons/WidgetsOutlined";
import Container from "~/components/Container";
import Hero from "~/components/Hero";
import InfoBox from "~/components/InfoBox";
import InfoBoxList from "~/components/InfoBoxList";
import { dedent } from "~/utils/dedentMultilineStrings";
import { Contact } from "./components/Contact";
import { ROUTE_ZFL_BEGLEITUNGEN, ROUTE_ZFL_TRAININGS } from "./routes";
import MetaTitle from "./ZFLMeta";

export default function ZFLIndex() {
  return (
    <>
      <MetaTitle />
      <Hero
        title="Bessere Rechtsetzung in allen Phasen Ihres Vorhabens"
        subtitle="Das Zentrum für Legistik unterstützt Sie bei der Ausarbeitung von Regelungen. Sie stellen damit sicher, dass Ihr Vorhaben betroffenenzentriert, praxistauglich und digital umsetzbar ist."
        className="bg-zfl-main20"
      />

      <div className="bg-yellow-500">
        <Container className="py-8">
          Das Angebot befindet sich aktuell im Aufbau und wird schrittweise mit
          Ihnen zusammen weiterentwickelt.
        </Container>
      </div>

      <Container className="py-40 md:py-80">
        <InfoBox
          visual={{
            type: "image",
            image: {
              url: "/zfl/Oval.png",
              alternativeText: "",
              size: "medium",
            },
          }}
          heading={{
            tagName: "h2",
            text: "Vorteile des Angebots für Sie",
          }}
          content={dedent`
            - Operative Hilfe beim Visualisieren
            - Unterstützung beim Erheben der Betroffenen-Bedürfnisse und Umsetzungshürden
            - Konkrete Formulierungsbeispiele und praktische Vorlagen
            - Vereinheitlichung von Checks und Arbeitshilfen
            - Praxisnahe Schulungen
            - Vernetzung und Austausch untereinander

            Gut zu wissen: Das Zentrum für Legistik ist ein Bestandteil des Handlungsfelds „Bessere Rechtsetzung“ in der Modernisierungsagenda.
          `}
        />
      </Container>

      <div className="bg-blue-100">
        <Container className="py-40 md:py-80">
          <InfoBoxList
            heading={{
              tagName: "h2",
              text: "Bereits verfügbare Angebote",
            }}
            items={[
              {
                look: "highlight",
                className: "bg-white",
                visual: {
                  type: "icon",
                  Icon: WidgetsOutlinedIcon,
                  className: "fill-blue-300",
                },
                heading: {
                  tagName: "h3",
                  text: "Interdisziplinäre Expertise für Ihr Regelungsvorhaben",
                },
                content:
                  "Wir begleiten Sie mit einem interdisziplinärem Kompetenzteam. Sie erhalten dadurch ein umfassendes Verständnis von Umsetzungsprozessen und gestalten Ihr Vorhaben praxisnah und bürokratiearm.",
                buttons: [
                  {
                    text: "Mehr Informationen zu interdisziplinären Kompetenzteams",
                    href: ROUTE_ZFL_BEGLEITUNGEN.href,
                    look: "link",
                  },
                ],
              },
              {
                look: "highlight",
                className: "bg-white",
                visual: {
                  type: "icon",
                  Icon: FactCheckOutlinedIcon,
                  className: "fill-blue-300",
                },
                heading: {
                  tagName: "h3",
                  text: "In Schulungen praktisches Wissen erlangen",
                },
                content:
                  "Neue Methoden und Werkzeuge für die Rechtsetzung in umsetzungsorientierten Schulungen kennenlernen.",
                buttons: [
                  {
                    text: "Mehr Informationen zu Schulungen",
                    href: ROUTE_ZFL_TRAININGS.href,
                    look: "link",
                  },
                ],
              },
            ]}
          />
        </Container>
      </div>

      <div>
        <Container className="py-40 md:py-80">
          <Contact className="bg-zfl-main20" />
        </Container>
      </div>
    </>
  );
}
