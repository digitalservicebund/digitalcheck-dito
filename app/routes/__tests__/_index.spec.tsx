import { render } from "@testing-library/react";
import { features } from "app/resources/features";
import Index from "app/routes/_index";
import { createRoutesStub } from "react-router";
import { describe, expect, it, vi } from "vitest";

vi.mock("react-router", async () => ({
  ...(await vi.importActual("react-router")),
  useOutletContext: () => ({
    featureFlags: Object.fromEntries(
      Object.entries(features).map(([_, value]) => [value, true]),
    ),
  }),
}));

const ResizeObserverMock = vi.fn(() => ({
  observe: vi.fn(),
  unobserve: vi.fn(),
  disconnect: vi.fn(),
}));

vi.stubGlobal("ResizeObserver", ResizeObserverMock);

const RouterStubIndex = createRoutesStub([
  {
    path: "/",
    Component: () => Index(),
  },
]);

describe("Landing page main content", () => {
  it("Renders correctly", () => {
    const { container } = render(<RouterStubIndex />);

    expect(container).toMatchSnapshot();
  });
});
