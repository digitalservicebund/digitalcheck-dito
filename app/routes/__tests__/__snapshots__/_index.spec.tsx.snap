// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Landing page main content > Renders correctly 1`] = `
<div>
  <div
    class="bg-blue-800 text-white"
  >
    <div
      class="container pt-40 pb-48 ds-stack ds-stack-16"
    >
      <h1
        class="hyphens-none"
      >
        Digitaltaugliche Regelungen erarbeiten
      </h1>
      <div
        class="ds-stack ds-stack-8 [&_a]:text-link ds-subhead mt-16"
      >
        <p>
          Der Digitalcheck ist eine Prozessbegleitung für Ihr Regelungsvorhaben. Sie
stellen damit Digitaltauglichkeit sicher.
        </p>
        

      </div>
    </div>
  </div>
  <div
    class="container py-40 lg:py-80"
  >
    <h2
      class="hyphens-none mb-40"
    >
      Schritt für Schritt
    </h2>
    <div
      class="relative"
    >
      <div
        class="absolute h-16 w-16 rounded-full bg-blue-300 max-lg:-bottom-8 max-lg:left-10 lg:-top-6 lg:-right-1"
      />
      <ol
        class="list-unstyled max-w-none max-lg:mx-16 lg:grid lg:grid-flow-col lg:grid-cols-3 lg:grid-rows-[repeat(6,auto)]"
      >
        <li
          class="relative border-blue-300 max-lg:border-l-3 max-lg:pb-40 max-lg:pl-40 lg:row-span-6 lg:grid lg:grid-rows-subgrid lg:border-t-3 lg:pr-40"
        >
          <div
            class="absolute flex h-40 w-40 items-center justify-center rounded-full bg-blue-800 font-bold text-white max-lg:-left-[22px] lg:-top-20"
          >
            1
          </div>
          <h3
            class="hyphens-none ds-heading-03-bold lg:mt-40"
          >
            Digitalbezug einschätzen
          </h3>
          <div
            class="mt-40 self-center"
          >
            <a
              class="ds-button"
              data-discover="true"
              href="/vorpruefung"
            >
               
              <span
                class="ds-button-label"
              >
                Digitalbezug einschätzen
              </span>
               
            </a>
          </div>
          <div
            class="mt-40"
          >
            <p
              class="font-bold"
            >
              Vorgehen
              :
            </p>
            <div
              class="ds-stack ds-stack-8 [&_a]:text-link"
            >
              <p>
                Sie identifizieren Digitalbezug und Interoperabilitätsanforderungen für Ihr Vorhaben.
              </p>
              

            </div>
          </div>
          <div
            class="mt-24 border-t-2 border-gray-400 pt-16"
          >
            <p
              class="font-bold"
            >
              Dauer
              :
            </p>
            <div
              class="ds-stack ds-stack-8 [&_a]:text-link"
            >
              <p>
                Wenige Minuten
              </p>
              

            </div>
          </div>
          <div
            class="mt-24"
          >
            <p
              class="font-bold"
            >
              Ergebnis
              :
            </p>
            <div
              class="ds-stack ds-stack-8 [&_a]:text-link"
            >
              <ul>
                

                <li>
                  Individueller Digital- und Interoperabilitätsbezug
                </li>
                

                <li>
                  Ergebnis als E-Mail zum Versenden an den Nationalen Normenkontrollrat
                </li>
                

              </ul>
              

            </div>
          </div>
        </li>
        <li
          class="relative border-blue-300 max-lg:border-l-3 max-lg:pb-40 max-lg:pl-40 lg:row-span-6 lg:grid lg:grid-rows-subgrid lg:border-t-3 lg:pr-40"
        >
          <div
            class="absolute flex h-40 w-40 items-center justify-center rounded-full bg-blue-800 font-bold text-white max-lg:-left-[22px] lg:-top-20"
          >
            2
          </div>
          <h3
            class="hyphens-none ds-heading-03-bold lg:mt-40"
          >
            Digitaltauglichkeit der Regelung sicherstellen
          </h3>
          <div
            class="mt-40 self-center"
          >
            <a
              class="ds-button ds-button-tertiary"
              data-discover="true"
              href="/methoden"
            >
               
              <span
                class="ds-button-label"
              >
                Regelung erarbeiten
              </span>
               
            </a>
          </div>
          <div
            class="mt-40"
          >
            <p
              class="font-bold"
            >
              Vorgehen
              :
            </p>
            <div
              class="ds-stack ds-stack-8 [&_a]:text-link"
            >
              <p>
                Bei einer positiven Vorprüfung wenden Sie die Werkzeuge und Methoden an, die die Digitaltauglichkeit Ihres Vorhabens sicherstellen.
              </p>
              

            </div>
          </div>
          <div
            class="mt-24 border-t-2 border-gray-400 pt-16"
          >
            <p
              class="font-bold"
            >
              Dauer
              :
            </p>
            <div
              class="ds-stack ds-stack-8 [&_a]:text-link"
            >
              <p>
                Punktuell über die gesamte Zeit der Regelungsausarbeitung
              </p>
              

            </div>
          </div>
          <div
            class="mt-24"
          >
            <p
              class="font-bold"
            >
              Ergebnis
              :
            </p>
            <div
              class="ds-stack ds-stack-8 [&_a]:text-link"
            >
              <ul>
                

                <li>
                  Ihre Regelung wird digitaltauglich
                </li>
                

                <li>
                  Visualisierungen zur digitaltauglichen Umsetzung Ihrer Regelung
                </li>
                

              </ul>
              

            </div>
          </div>
        </li>
        <li
          class="relative border-blue-300 max-lg:border-l-3 max-lg:pb-40 max-lg:pl-40 lg:row-span-6 lg:grid lg:grid-rows-subgrid lg:border-t-3 lg:pr-40"
        >
          <div
            class="absolute flex h-40 w-40 items-center justify-center rounded-full bg-blue-800 font-bold text-white max-lg:-left-[22px] lg:-top-20"
          >
            3
          </div>
          <h3
            class="hyphens-none ds-heading-03-bold lg:mt-40"
          >
            Dokumentieren der Digitaltauglichkeit
          </h3>
          <div
            class="mt-40 self-center"
          >
            <a
              class="ds-button ds-button-tertiary"
              data-discover="true"
              href="/dokumentation"
            >
               
              <span
                class="ds-button-label"
              >
                Dokumentation erstellen
              </span>
               
            </a>
          </div>
          <div
            class="mt-40"
          >
            <p
              class="font-bold"
            >
              Vorgehen
              :
            </p>
            <div
              class="ds-stack ds-stack-8 [&_a]:text-link"
            >
              <p>
                Sie dokumentieren, wie Sie Digitaltauglichkeit in Ihrer Regelung sichergestellt haben.
              </p>
              

            </div>
          </div>
          <div
            class="mt-24 border-t-2 border-gray-400 pt-16"
          >
            <p
              class="font-bold"
            >
              Dauer
              :
            </p>
            <div
              class="ds-stack ds-stack-8 [&_a]:text-link"
            >
              <p>
                Ein bis zwei Stunden
              </p>
              

            </div>
          </div>
          <div
            class="mt-24"
          >
            <p
              class="font-bold"
            >
              Ergebnis
              :
            </p>
            <div
              class="ds-stack ds-stack-8 [&_a]:text-link"
            >
              <ul>
                

                <li>
                  Ausgefülltes Word-Dokument, welches Sie dem Nationalen Normenkontrollrat senden
                </li>
                

                <li>
                  Ihr Vorhaben ist formal bereit für die Ressortabstimmung
                </li>
                

              </ul>
              

            </div>
          </div>
        </li>
      </ol>
    </div>
  </div>
  <div
    class="bg-blue-100"
  >
    <div
      class="container ds-stack ds-stack-40 py-40 lg:py-80"
    >
      <h2
        class="hyphens-none"
      >
        Grundlagen zum Digitalcheck
      </h2>
      <div
        class="flex scroll-my-40 gap-32 md:flex-row flex-col rounded-lg px-16 py-40 sm:px-80 bg-white"
        data-testid="info-box-container"
      >
        <svg
          aria-hidden="true"
          class="hidden fill-blue-500 sm:block size-80"
          data-testid="WidgetsOutlinedIcon"
          focusable="false"
          height="24"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 0h24v24H0V0z"
            fill="none"
          />
          <path
            d="M16.66 4.52l2.83 2.83-2.83 2.83-2.83-2.83 2.83-2.83M9 5v4H5V5h4m10 10v4h-4v-4h4M9 15v4H5v-4h4m7.66-13.31L11 7.34 16.66 13l5.66-5.66-5.66-5.65zM11 3H3v8h8V3zm10 10h-8v8h8v-8zm-10 0H3v8h8v-8z"
          />
        </svg>
        <div
          class="ds-stack ds-stack-16 break-words"
          data-testid="info-box-content"
        >
          <h3
            class="hyphens-none"
          >
            Was ist Digitaltauglichkeit? Und wie unterstützt Sie der Digitalcheck dabei?
          </h3>
          <div
            class="ds-stack ds-stack-8 [&_a]:text-link"
          >
            <p>
              Digitaltaugliche Regelungen ermöglichen eine rechtlich und technisch reibungslose digitale Umsetzung und fördern die automatisierte Bearbeitung von Verfahrensschritten.
            </p>
            

          </div>
          <div
            class="ds-stack ds-stack-8"
          >
            <ul
              class="list-unstyled flex flex-col gap-8"
            >
              <li>
                 
                <a
                  class="text-link increase-tap-area flex items-center"
                  data-discover="true"
                  href="/grundlagen/digitaltauglichkeit"
                >
                  Mehr zur Digitaltauglichkeit erfahren
                   
                </a>
              </li>
            </ul>
          </div>
        </div>
      </div>
      <div
        class="flex scroll-my-40 gap-32 md:flex-row flex-col rounded-lg px-16 py-40 sm:px-80 bg-white"
        data-testid="info-box-container"
      >
        <svg
          aria-hidden="true"
          class="hidden fill-blue-500 sm:block size-80"
          data-testid="FactCheckOutlinedIcon"
          enable-background="new 0 0 24 24"
          focusable="false"
          height="24"
          viewBox="0 0 24 24"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <g>
            <rect
              fill="none"
              height="24"
              width="24"
            />
          </g>
          <g>
            <g>
              <path
                d="M20,3H4C2.9,3,2,3.9,2,5v14c0,1.1,0.9,2,2,2h16c1.1,0,2-0.9,2-2V5 C22,3.9,21.1,3,20,3z M20,19H4V5h16V19z"
                fill-rule="evenodd"
              />
              <polygon
                fill-rule="evenodd"
                points="19.41,10.42 17.99,9 14.82,12.17 13.41,10.75 12,12.16 14.82,15"
              />
              <rect
                fill-rule="evenodd"
                height="2"
                width="5"
                x="5"
                y="7"
              />
              <rect
                fill-rule="evenodd"
                height="2"
                width="5"
                x="5"
                y="11"
              />
              <rect
                fill-rule="evenodd"
                height="2"
                width="5"
                x="5"
                y="15"
              />
            </g>
          </g>
        </svg>
        <div
          class="ds-stack ds-stack-16 break-words"
          data-testid="info-box-content"
        >
          <h3
            class="hyphens-none"
          >
            Worauf achtet der Nationale Normenkontrollrat?
          </h3>
          <div
            class="ds-stack ds-stack-8 [&_a]:text-link"
          >
            <p>
              Der Nationale Normenkontrollrat (NKR) prüft das Vorhaben auf Digitaltauglichkeit. Mit der Anwendung der 
              <strong>
                Prinzipien
              </strong>
               und Erstellung von 
              <strong>
                Visualisierungen
              </strong>
               kann der Nationale Normenkontrollrat die Digitaltauglichkeit besser und schneller nachvollziehen.
            </p>
            

          </div>
          <div
            class="ds-stack ds-stack-8"
          >
            <ul
              class="list-unstyled flex flex-col gap-8"
            >
              <li>
                 
                <a
                  class="text-link increase-tap-area flex items-center"
                  data-discover="true"
                  href="/grundlagen/normenkontrollrat"
                >
                  Mehr zum NKR erfahren
                   
                </a>
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="bg-[url('/images/trainings.jpg')] bg-cover bg-center"
  >
    <div
      class="container py-40 lg:py-80"
    >
      <div
        class="flex flex-col gap-24 sm:flex-row"
      >
        <div
          class="flex scroll-my-40 gap-32 md:flex-row flex-col px-16 py-32 sm:px-32 sm:pt-40 sm:pb-48 bg-white"
          data-testid="info-box-container"
        >
          <div
            class="ds-stack ds-stack-16 break-words"
            data-testid="info-box-content"
          >
            <h3
              class="hyphens-none"
            >
              Das Potenzial von Visualisierungen nutzen
            </h3>
            <div
              class="ds-stack ds-stack-8 [&_a]:text-link"
            >
              <p>
                Visualisierungen helfen bestehende Abläufe Ihrer Regelung einfach sichtbar zu machen und Digitaltauglichkeit einzuschätzen.
              </p>
              

            </div>
            <div
              class="flex flex-wrap gap-16"
            >
              <a
                class="ds-button ds-button-tertiary"
                data-discover="true"
                href="/methoden/ablaeufe-aufgaben-erfassen"
              >
                 
                <span
                  class="ds-button-label"
                >
                  Zu Visualisierungen
                </span>
                 
              </a>
              <a
                class="ds-button ds-button-ghost"
                data-discover="true"
                href="/beispiele/visualisierungen"
              >
                 
                <span
                  class="ds-button-label"
                >
                  Beispiele
                </span>
                 
              </a>
            </div>
          </div>
        </div>
        <div
          class="flex scroll-my-40 gap-32 md:flex-row flex-col px-16 py-32 sm:px-32 sm:pt-40 sm:pb-48 bg-white"
          data-testid="info-box-container"
        >
          <div
            class="ds-stack ds-stack-16 break-words"
            data-testid="info-box-content"
          >
            <h3
              class="hyphens-none"
            >
              Prinzipien als Grundlage für Digitaltauglichkeit einsetzen
            </h3>
            <div
              class="ds-stack ds-stack-8 [&_a]:text-link"
            >
              <p>
                Fünf Prinzipien helfen Ihnen dabei, Chancen der digitalen Umsetzung auszuschöpfen und Hindernisse zu erkennen.
              </p>
              

            </div>
            <div
              class="flex flex-wrap gap-16"
            >
              <a
                class="ds-button ds-button-tertiary"
                data-discover="true"
                href="/methoden/fuenf-prinzipien"
              >
                 
                <span
                  class="ds-button-label"
                >
                  Zu den Prinzipien
                </span>
                 
              </a>
              <a
                class="ds-button ds-button-ghost"
                data-discover="true"
                href="/beispiele/prinzipien"
              >
                 
                <span
                  class="ds-button-label"
                >
                  Beispiele
                </span>
                 
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="container space-y-48 py-40 lg:py-80"
  >
    <div
      class="flex scroll-my-40 gap-32 md:flex-row flex-col"
      data-testid="info-box-container"
    >
      <div
        class="ds-stack ds-stack-16 break-words"
        data-testid="info-box-content"
      >
        <h2
          class="hyphens-none"
        >
          Individuelle Unterstützung für Ihr Vorhaben
        </h2>
        <div
          class="ds-stack ds-stack-8 [&_a]:text-link"
        >
          <p>
            Nutzen Sie unsere persönlichen Unterstützungsangebote, um Ihr Regelungsvorhaben digitaltauglich zu gestalten und den Prozess erfolgreich zu durchlaufen.
          </p>
          

        </div>
        <div
          class="flex flex-wrap gap-16"
        >
          <a
            class="ds-button ds-button-tertiary"
            data-discover="true"
            href="/unterstuetzung"
          >
             
            <span
              class="ds-button-label"
            >
              Angebote kennenlernen
            </span>
             
          </a>
        </div>
      </div>
    </div>
    <div
      class="flex scroll-my-40 gap-32 md:flex-row flex-col rounded-lg px-16 py-40 sm:px-80 bg-blue-100"
      data-testid="info-box-container"
    >
      <svg
        aria-hidden="true"
        class="hidden fill-blue-500 sm:block size-80"
        data-testid="ContactPhoneOutlinedIcon"
        focusable="false"
        height="24"
        viewBox="0 0 24 24"
        width="24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M0 0h24v24H0V0z"
          fill="none"
        />
        <path
          d="M22 3H2C.9 3 0 3.9 0 5v14c0 1.1.9 2 2 2h20c1.1 0 1.99-.9 1.99-2L24 5c0-1.1-.9-2-2-2zm0 16H2V5h20v14zm-2.99-1.01L21 16l-1.51-2h-1.64c-.22-.63-.35-1.3-.35-2s.13-1.37.35-2h1.64L21 8l-1.99-1.99c-1.31.98-2.28 2.37-2.73 3.99-.18.64-.28 1.31-.28 2s.1 1.36.28 2c.45 1.61 1.42 3.01 2.73 3.99zM9 12c1.65 0 3-1.35 3-3s-1.35-3-3-3-3 1.35-3 3 1.35 3 3 3zm0-4c.55 0 1 .45 1 1s-.45 1-1 1-1-.45-1-1 .45-1 1-1zm6 8.59c0-2.5-3.97-3.58-6-3.58s-6 1.08-6 3.58V18h12v-1.41zM5.48 16c.74-.5 2.22-1 3.52-1s2.77.49 3.52 1H5.48z"
        />
      </svg>
      <div
        class="ds-stack ds-stack-16 break-words"
        data-testid="info-box-content"
      >
        <h3
          class="hyphens-none ds-heading-03-reg mb-16"
        >
          „Der Digitalcheck erscheint mit den Hilfestellungen unkomplizierter als gedacht.“
        </h3>
        <div
          class="ds-stack ds-stack-8 [&_a]:text-link"
        >
          <p>
            <strong>
              Referentin
            </strong>
            <br />
            aus einem Bundesministerium
          </p>
          

        </div>
      </div>
    </div>
  </div>
</div>
`;
